docs/contribution/nearcore.md

```diff
@@ -114,3 +114,108 @@ Official image is published at `nearprotocol/nearcore`
+real	5m57.307s user	37m28.079s sys	2m27.664s
+# touch */*/*/*.rs; time RUSTFLAGS='-C lto=off -C link-arg=-fuse-ld=lld' RUSTC_WRAPPER= cargo build -p neard --release
+real	1m26.627s user	7m12.705s sys	0m7.900s
+
+```
+
+# experimental share-generics feature
+```bash
+# time RUSTFLAGS='-C lto=off -C link-arg=-fuse-ld=lld -Zshare-generics=y' RUSTC_WRAPPER= cargo build -p neard --release
+real	4m35.538s user	29m17.550s sys	2m6.262s
+# touch */*/*/*.rs; time RUSTFLAGS='-C lto=off -C link-arg=-fuse-ld=lld -Zshare-generics=y' RUSTC_WRAPPER= cargo build -p neard --release
+real	0m53.101s user	3m52.850s sys	0m7.317s
+```
+
+# parallelize building invidual cargo packages
+```bash
+# time RUSTFLAGS='-C lto=off -C link-arg=-fuse-ld=lld -Zshare-generics=y -C codegen-units=6' RUSTC_WRAPPER= cargo build -p neard --release
+real	4m42.485s user	28m47.223s sys	2m18.833s
+# touch */*/*/*.rs; time RUSTFLAGS='-C lto=off -C link-arg=-fuse-ld=lld -Zshare-generics=y -C codegen-units=6' RUSTC_WRAPPER= cargo build -p neard --release
+real	0m51.620s user	4m13.180s sys	0m11.254s
+```
+
+# cache results from previous compilations using sccache
+```bash
+# time RUSTFLAGS='-C lto=off -C link-arg=-fuse-ld=lld -Zshare-generics=y -C codegen-units=6' RUSTC_WRAPPER=sccache cargo build -p neard --release
+real	4m1.409s user	5m48.943s sys	0m51.071s
+# time RUSTFLAGS='-C lto=off -C link-arg=-fuse-ld=lld -Zshare-generics=y -C codegen-units=6' RUSTC_WRAPPER=sccache cargo build -p neard --release
+real	1m14.038s user	5m25.992s sys	0m57.247s
+```
+
+## Recommended settings
+
+# Setting for building release binary without debug symbols
+```bash
+# time RUSTFLAGS='-C lto=off -C link-arg=-fuse-ld=lld -Zshare-generics=y -C codegen-units=6' RUSTC_WRAPPER=sccache cargo build -p neard --release
+real	4m1.409s user	5m48.943s sys	0m51.071s
+# time RUSTFLAGS='-C lto=off -C link-arg=-fuse-ld=lld -Zshare-generics=y -C codegen-units=6' RUSTC_WRAPPER=sccache cargo build -p neard --release
+real	1m14.038s user	5m25.992s sys	0m57.247s
+# touch */*/*/*.rs; time RUSTFLAGS='-C lto=off -C link-arg=-fuse-ld=lld -Zshare-generics=y -C codegen-units=6' RUSTC_WRAPPER=sccache cargo build -p neard --release
+real	0m7.371s user	0m5.646s sys	0m1.763s
+
+```
+
+# Setting for building release binary with debug symbols and reduced inlining
+```bash
+# time RUSTFLAGS='-C lto=off -C link-arg=-fuse-ld=lld -Zshare-generics=y -C codegen-units=6 -C inline-threshold=25 -C debuginfo=2' RUSTC_WRAPPER=sccache cargo build -p neard --release
+real	4m3.216s user	6m2.943s sys	0m54.196s
+# time RUSTFLAGS='-C lto=off -C link-arg=-fuse-ld=lld -Zshare-generics=y -C codegen-units=6 -C inline-threshold=25 -C debuginfo=2' RUSTC_WRAPPER=sccache cargo build -p neard --release
+real	1m30.039s user	5m50.652s sys	1m2.509s
+# touch */*/*/*.rs; time RUSTFLAGS='-C lto=off -C link-arg=-fuse-ld=lld -Zshare-generics=y -C codegen-units=6 -C inline-threshold=25 -C debuginfo=2' RUSTC_WRAPPER=sccache cargo build -p neard --release
```

Comment:
seems that there is no difference in these commands?

Reply:
fixed